### ISSUE TITLE
Atom is not saving

### ISSUE THREAD

**Issue by arimozes** [atom/atom#18393]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
    * Followed all applicable steps in the debugging guide: https://flight-manual.atom.io/hacking-atom/sections/debugging/
    * Checked the FAQs on the message board for common solutions: https://discuss.atom.io/c/faq
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom
    * Checked that there is not already an Atom package that provides the described functionality: https://atom.io/packages

### Description

This is the same issue as reported [here](https://github.com/atom/atom/issues/18114), which was automatically closed due to lack of response from the original filer.
I have the exact same problem as @romdoq mentioned in that issue comments section.

### Steps to Reproduce

1) Open atom (with or without safe mode).
2) Edit a file and save - works ok.
3) Edit a second time and save - works ok.
4) Edit a third time and save - blue dot remains and file does not get saved. If I try to close atom - it indicates there is content that needs to be saved.  The issue always appears to happen as of the third save request.

Try to open a file at this point (ctrl-o or via menu) does not do anything.
Restarting atom allows things to work once again for a moment, but then fails as above.
Disabling github integration fixes the problem as described by @romdoq .
Starting atom as root (sudo atom) does not exhibit the issue - so maybe a permission related issue (perhaps with git) ?

**Expected behavior:** [What you expect to happen]
I expect the blue dot to disappear and the save to complete.

**Actual behavior:** [What actually happens]
File is not saved.

**Reproduces how often:** [What percentage of the time does it reproduce?]
Always, unless I disable github integration or I start up atom with root privs (sudo atom)

### Versions

$ atom --version
Atom : 1.32.1
Electron: 2.0.9
Chrome : 61.0.3163.100
Node : 8.9.3

$ git --version
git version 2.7.4

Running on Ubuntu 16.04

### Additional Information

Any additional information, configuration or data that might be necessary to reproduce the issue.

- Are you a new user or seasoned Atom user? ..when did you notice this behaviour?
I am a new user of Atom, but a seasoned software developer.
 

- Does this type of behaviour happen for all file type? ..does it do it for TXT files, for example?
Yes.

- Could you check the time stamp of the file that you ordered to be saved. Does the file's time stamp change if you order the file to save?
It does not change.
 

- Could you indicate which packages you have? ..please run apm ls from the command prompt
All built-in packages (other than github integration to workaround the problem), and no community packages.

- Additionally, do you see any errors in devtools console?
No errors appear in the console

- Local or remote file system? Are you sure you have proper permissions?
Local.  Yes, I checked through various files and directories (including .atom and .config/Atom) and didn't come across anything suspicious.

Let me know if I can provide additional information.

**Comment by rsese** [comment]
Thanks for the report!

> Starting atom as root (sudo atom) does not exhibit the issue - so maybe a permission related issue (perhaps with git) ?

Starting Atom with `sudo` can cause problems, see https://discuss.atom.io/t/session-saved-when-sudo-how-to-when-not-sudo/38253 for example.  Can you make sure your Atom home directory and its contents are owned by your user (where `USER` and `GROUP` are your account's user and group settings)?

```
sudo chown -R USER:GROUP ~/.atom
```

**Comment by arimozes** [comment]
First, thanks for the quick response.
Wish I copied out the directory before running the chown just to figure out the root cause.
I did the chown and I cannot reproduce any longer.  However, I had fully deleted the .atom and .config/Atom directories multiple times before, and that did not address the issue, so I am not sure why it would have been an issue with the privs in there...
In any case, since it appears I can no longer reproduce, I will close and happily continue using Atom.
